### DISCUSSION TITLE
在Mac下搭建好了之后无论如何都连不上

### DISCUSSION THREAD

**Discussion by Etern11ty** [shadowsocks/shadowsocks-windows#3410] [Q&A]
<img width="570" alt="image" src="https://github.com/shadowsocks/shadowsocks-windows/assets/134341061/31d1df18-a645-40c5-b082-30cb0ea03c6a">
启动之后不管是手机还是iPad都无法进行连接，报错显示解析失败

**Comment by sgf** [comment]
443?端口,你是认真的么
